bin/apisix

```diff
@@ -560,6 +560,23 @@ local function init_etcd(show_output)
     for index, host in ipairs(yaml_conf.etcd.host) do
         local is_success = true
 
+        local token_head = ""
+        if etcd_conf.user and etcd_conf.password then
+            local uri_auth = host .. "/v3/auth/authenticate"
+            local post_json_auth = '{"name": "' .. etcd_conf.user .. '", "password": "' .. etcd_conf.password .. '"}'
```

Comment:
Could we use some libraries or functions to stringify that object instead of combining string?

https://stackoverflow.com/questions/24908199/convert-json-string-to-lua-table

Reply:
According to https://github.com/apache/apisix/pull/2690/files#diff-0cfc3cc4673fc5c2de7be20b9f8763a8955212292f08a8a7b5d1f459533f5fcdR572 , could we use `dkjson.encode`?

Reply:
yes, `dkjson` is more safe.

Reply:
Done